viper.go

```diff
@@ -879,9 +885,12 @@ func (v *Viper) BindEnv(input ...string) error {
 // Given a key, find the value.
 // Viper will check in the following order:
 // flag, env, config file, key/value store, default.
-// Viper will check to see if an alias exists first.
-// Note: this assumes a lower-cased key given.
-func (v *Viper) find(lcaseKey string) interface{} {
+// Viper will then check in the following order:
+// flag, env, config file, key/value store.
+// Lastly, if no value was found and flagDefault is true, and if the key
+// corresponds to a flag, the flag's default value is returned.
+//
+func (v *Viper) find(lcaseKey string, flagDefault bool) interface{} {
```

Comment:
@losinggeneration `flagDefault` is not being used anywhere.

Reply:
> This has code partially based on #331 for b7a4909 that uses the same flag name & updated comment for flag. Though the comment isn't totally correct for this PR, but #331 IMO should be merged.

I probably shouldn't have pulled that along from #331 for that commit. I can revert that part since it's not used here.